pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -300,21 +304,20 @@ def test_merge_single_cells():
     )
 
     # Confirm that the merge correctly reversed the object number (opposite from Parent)
-    assert (
-        sc_merged_df.Metadata_ObjectNumber_cytoplasm.tolist()[::-1]
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
-    )
-    assert (
-        manual_merge.Metadata_ObjectNumber_cytoplasm.tolist()[::-1]
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
-    )
-    assert (
-        manual_merge.Metadata_ObjectNumber_cytoplasm.tolist()[::-1]
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
-    )
-    assert (
-        manual_merge.Metadata_ObjectNumber_cells.tolist()
-        == sc_merged_df.Metadata_ObjectNumber.tolist()
+    assert_cols = [
+        "Metadata_ObjectNumber",
+        "Metadata_ObjectNumber_cytoplasm",
+        "Metadata_ObjectNumber_cells",
+    ]
+    # check that we have the same data using same cols, sort and a reset index
+    pd.testing.assert_frame_equal(
+        left=manual_merge[assert_cols]
+        .sort_values(by=assert_cols, ascending=True)
```

Comment:
I am a bit concerned that sorting is giving us undue confidence of equivalent results. Am I reading this wrong? How should I be thinking about this?

Can you add an explicit check that the object number is the same? Oh, but maybe this code block is doing exactly that (and more!)?

Reply:
Great comment, thank you! The chunked merges made the data inconsistent with the existing tests. The check here now looks at columns "Metadata_ObjectNumber", "Metadata_ObjectNumber_cytoplasm", "Metadata_ObjectNumber_cells", sorted by the same columns in the same order for data equivalency and does include an Metadata_ObjectNumber check as a result. 

Would users of `merge_single_cells` expect for sorted output by the same columns or similar? We could include this as an additional step within the method, which would likely avoid needing to do that manually within testing here.